datajoint/fetch.py

```diff
@@ -20,46 +25,57 @@ def prepare_attributes(relation, item):
         raise DataJointError("Index must be a slice, a tuple, a list, a string.")
     return item, attributes
 
-class FetchQuery:
+def copy_first(f):
+    @wraps(f)
+    def ret(*args, **kwargs):
+        args = list(args)
+        args[0] = args[0].__class__(args[0]) # call copy constructor
+        return f(*args, **kwargs)
 
-    def __init__(self, relation):
-        """
+    return ret
 
-        """
-        self.behavior = dict(
-            offset=0, limit=None, order_by=None, descending=False, as_dict=False, map=None
-        )
-        self._relation = relation
+class Fetch:
+    def __init__(self, relation):
+        if isinstance(relation, Fetch): # copy constructor
+            self.behavior = dict(relation.behavior)
+            self._relation = relation._relation
+        else:
+            self.behavior = dict(
+                offset=0, limit=None, order_by=None, as_dict=False
+            )
+            self._relation = relation
 
 
+    @copy_first
     def from_to(self, fro, to):
         self.behavior['offset'] = fro
```

Comment:
That's in line with the index convention of python and I think it makes a lot of sense. If you want to split an interval a,b at c you can say from_to(a,c) and from_to(c,b) instead of from_to(a,c) and from_to(c+1,b).